practice-two/src/style/sections/review.css

```diff
@@ -0,0 +1,159 @@
+.review {
+  padding-top: 28px;
+  padding-bottom: 48px;
+}
+
+.review__wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.review__card.review__card--light {
+  display: none;
+}
+
+.review__card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-right: 25px;
+  padding-left: 25px;
+  padding-bottom: 70px;
+  border-radius: 5px;
+  box-shadow: 0px 5px 15px var(--box-shadow-color);
+  margin-top: 28px;
+
+  &::before {
+    content: '"';
+    font-size: var(--font-size-3xlg);
+    font-style: var(--font-style-default);
+    font-weight: var(--font-weight-primary);
+    line-height: var(--line-height-normal);
+    padding-top: 20px;
+  }
+}
+
+.review__card--platinum {
+  background-color: var(--color-secondary);
+
+  &::before {
+    color: var(--color-base);
+  }
+}
+
+.review__card-description {
+  width: 190px;
+}
+
+.review__card-name {
+  font-size: var(--font-size-md);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+  line-height: var(--line-height-normal);
+  margin-top: 27px;
+}
+
+.review__person-img {
+  position: absolute;
+  bottom: -10%;
+  width: 64px;
+  height: 64px;
+}
+
+.review__icon {
+  margin-top: 50px;
+}
+
+.card__position {
+  font-size: var(--font-size-xs);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+  line-height: var(--line-height-2xlg);
+}
+
+.review__icon-dot {
+  width: 8px;
+  height: 8px;
+  fill: none;
+
+  &:hover {
+    fill: var(--color-secondary);
+    cursor: pointer;
+  }
+}
+
+.review__icon {
+  display: flex;
+  gap: 22px;
+  margin-top: 40px;
+}
+
+@media (min-width: 768px) {
+  .dotnav-mobile-icon {
+    display: none;
+  }
+
+  .review__person-img {
+    left: 60%;
+  }
+
+  .review__content {
+    display: flex;
+    gap: 30px;
+    margin-top: 32px;
+  }
+
+  .review__card--light {
+    background-color: var(--color-primary);
+  }
+
+  .review__card {
+    padding-right: 63px;
+    padding-left: 63px;
+  }
+
+  .review__card.review__card--light {
+    display: flex;
+  }
+
+  .dot-icon-hidden {
+    display: none;
+  }
+}
+
+@media (min-width: 1440px) {
+  .review__card-description {
+    width: 420px;
+  }
+
+  .review__person-img {
+    bottom: -20%;
+    width: 128px;
+    height: 128px;
+  }
+
+  .review__card {
+    padding: 10px 67px 80px;
+  }
+
+  .review__card-name {
+    --font-size: 24px;
+    font-size: var(--font-size);
```

Comment:
I see that you have use rem for font size in `variable.css` file. Why don't you use rem here?